pkg/blockstorage/gcepd/gcepd.go

```diff
@@ -111,7 +110,7 @@ func (s *GpdStorage) VolumeCreate(ctx context.Context, volume blockstorage.Volum
 
 	createDisk := &compute.Disk{
 		Name:   fmt.Sprintf(volumeNameFmt, uuid.NewV1().String()),
-		SizeGb: volume.Size,
+		SizeGb: blockstorage.SizeInG(volume.SizeInBytes),
```

Comment:
This GCE doc states that they use GiB: https://cloud.google.com/compute/disks-image-pricing.

I quote:
```
...
For Compute Engine, disk size, machine type memory, and network usage are calculated in gigabytes (GB), where 1 GB is 230 bytes. This unit of measurement is also known as a gibibyte (GiB).
...
```